### ISSUE TITLE
InfluxDB: database "telegraf" creation failed: 403 Forbidden

### ISSUE THREAD

**Issue by sgofferj** [sabuto/hassio-telegraf#52]
#Type of Issue
Issue

#Description of issue
Telegraf tries to create telegraf DB which already exists

#Ideas to fix
Add `skip_database_creation = true` to config, either fixed or configurable.

# Logs
2022-09-16T06:28:21Z W! [outputs.influxdb] When writing to [https://influx]: database "telegraf" creation failed: 403 Forbidden
panic: runtime error: slice bounds out of range [:4] with capacity 0

**Comment by SNoof85** [comment]
You can do this with the custom config located in /share/telegraf.conf and update the `skip_database_creation`to true (uncomment it  !) and load that custom config with :
```
enabled: true
location: /share/telegraf.conf
``` in the addon configuration

**Comment by sgofferj** [comment]
> You can do this with the custom config located in /share/telegraf.conf and update the `skip_database_creation`to true (uncomment it  !) and load that custom config with :
> ```
> enabled: true
> location: /share/telegraf.conf
> ``` 
> in the addon configuration

Yes, I know. But this issue makes the addon work only with a custom configuration, rendering the GUI config useless.
